selfdrive/manager/process_config.py

```diff
@@ -69,6 +69,7 @@ def qcomgps(started, params, CP: car.CarParams) -> bool:
   PythonProcess("updated", "selfdrive.updated", enabled=not PC, onroad=False, offroad=True),
   PythonProcess("uploader", "system.loggerd.uploader", offroad=True),
   PythonProcess("statsd", "selfdrive.statsd", offroad=True),
+  PythonProcess("webserver", "system.webserver.webserverd", offroad=True),
```

Comment:
```suggestion
  PythonProcess("webserver", "system.webserver.webserverd", onroad=False, offroad=True),
```
Personally, I would prefer to have this process run offroad only to save resources.